tests/README.rst

```diff
@@ -80,6 +80,159 @@ To do that, run `podman pull quay.io/rhinstaller/anaconda-ci:master` or build
 it locally again.
 
 
+GH action integration
+---------------------
+
+This section explains our GH actions solution for testing pull requests to Anaconda.
+Explanation of parts are below.
+
+
+Explanation of the GH configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+master:
+^^^^^^^
+
+Everything is stored in ``.github/workflows/*.yml`` configuration files. These
+files completely describe what steps are required to run the tests, what are the
+triggers and so on. We use GH runners here so we don't have to care too much
+about what is executed there. GH runners are designed to be isolated and secure.
+On master we are building the container only if the container files have changed
+otherwise we are pulling the container from our repository. For more
+information see below.
+
+RHEL:
+^^^^^
+
+Again configuration is stored in ``.github/workflows/*.yml`` files. However,
+here we also have (fully automatically deployed) runners in our Upshift instance.
+GH configuration is using our self-hosted runners. These runners are containers
+with all the dependencies in place so the yml configuration will just execute
+tests. That means that part of the configuration is in our builders repo (private)
+and that is how we are creating the runners. However, you can also start runners
+locally by running the container and providing GH token. That is pretty valuable
+in case of workflow testing. See Dockerfile on your selected branch for more
+details.
+
+To protect our self-hosted runners all the external contributions have to be
```

Comment:
To protect our self-hosted runners, tests only run automatically for [rhinstaller organization members](https://github.com/orgs/rhinstaller/people). For external contributors, an organization member needs to approve the test run by sending a comment starting with `/tests`.